src/rad/appdata.cpp

```diff
@@ -1323,24 +1324,15 @@ bool ApplicationData::LoadProject( const wxString &file, bool justGenerate )
 				wxLogError( wxT( "This project file is out of date.  Update your .fbp before using --generate" ) );
 				return false;
 			}
-			if ( wxYES == wxMessageBox( wxT( "This project file is not of the current version.\n" )
-			                            wxT( "Would you to attempt automatic conversion?\n\n" )
-			                            wxT( "NOTE: This will modify your project file on disk!" ), _( "Old Version" ), wxYES_NO ) )
-			{
-				// we make a backup of the project
-				::wxCopyFile( file, file + wxT( ".bak" ) );
 
-				if ( !ConvertProject( file, fbpVerMajor, fbpVerMinor ) )
-				{
-					wxLogError( wxT( "Unable to convert project" ) );
-					return false;
-				}
+			wxMessageBox(
+			    _("This project is using an older file format. It will be converted in memory while loading it.\n\n"
+			      "Caution! If you save the project, it will be overwritten with the newer file format."),
```

Comment:
Better than before, but somehow i am still not happy about it. How about something like this:

This project file is using an older file format, it will be updated during loading.
WARNING: Saving the project will update the format of the project file on disk!